tasks/install.yml

```
@@ -1,18 +1,22 @@
 ---
-- name: Ensure apt key is present
-  tags: [ install ]
-  ansible.builtin.apt_key:
-    url: 'https://www.aptly.info/pubkey.txt'
-    state: present
+- name: Add aptly repository
+  when:
+    - aptly__add_repo is true
```

Can be simplified, no need to compare a boolean.

```suggestion
    - aptly__add_repo
```